src/Network/Response.php

```diff
@@ -56,7 +56,7 @@ class Response
         303 => 'See Other',
         304 => 'Not Modified',
         305 => 'Use Proxy',
-        306 => 'Switch Proxy',
+        306 => '(Unused)',
```

Comment:
So the question remains why add it to the list when this code is not supposed to be used?